docs/sources/alerting/set-up/provision-alerting-resources/_index.md

```diff
@@ -33,23 +33,26 @@ Choose from the options below to import (or provision) your Grafana Alerting res
 1. [Use configuration files to provision your alerting resources](/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/provision-alerting-resources/file-provisioning), such as alert rules and contact points, through files on disk.
 
    {{< admonition type="note" >}}
-   File provisioning is not available in Grafana Cloud instances.
-   {{< /admonition >}}
+
+   - You cannot edit provisioned resources from files in the Grafana UI.
+   - Provisioning with configuration files is not available in Grafana Cloud.
+     {{< /admonition >}}
```

Comment:
The indentation of the closing admonition tag looks wrong here but it seems to be consistent with what's in `main` and it looks fine in https://grafana.com/docs/grafana/next/alerting/set-up/provision-alerting-resources/